src/_zkapauthorizer/tests/test_tahoe.py

```diff
@@ -204,34 +204,14 @@ class DirectoryTestsMixin:
     # Support test methods that return a Deferred.
     run_tests_with = AsynchronousDeferredRunTest.make_factory(timeout=60.0)
 
-    @inlineCallbacks
-    def test_make_directory(self):
-        """
-        ``make_directory`` returns a coroutine that completes with the capability
-        of a new, empty directory.
-        """
-        tahoe = self.get_client()
-
-        dir_cap = yield Deferred.fromCoroutine(tahoe.make_directory())
-
-        # If we can download it, consider that success.
-        outpath = FilePath(self.useFixture(TempDir()).join("dir_contents"))
-        yield Deferred.fromCoroutine(tahoe.download(outpath, dir_cap, None))
-        self.assertThat(outpath.getContent(), Not(Equals(b"")))
```

Comment:
This test broke because I disabled downloading directory caps.  You have to `list_directory` them now - which is just what the next test does.